pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -138,7 +138,8 @@ def write_pep0(self, peps: list[PEP]):
             ("Abandoned, Withdrawn, and Rejected PEPs", dead),
         ]
         for (category, peps_in_category) in pep_categories:
-            self.emit_pep_category(category, peps_in_category)
+            if is_subindex and len(peps_in_category) > 0:
```

Comment:
```suggestion
            if not is_subindex or len(peps_in_category) > 0:
```

It seems this logic may have been mistaken, as this is why the subcategories aren't getting generated at all anymore on PEP 0.